validator/testdata/feature_tests/amp-image-slider.html

```diff
@@ -53,5 +53,30 @@
 
   <div first>Should not be exposed</div>
   <div second>Should not be exposed</div>
+
+  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" disable-hint-reappear>
+    <amp-img src="/1.jpg" layout="fill"></amp-img>
+    <amp-img src="/2.jpg" layout="fill"></amp-img>
+  </amp-image-slider>
+
+  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" initial-slider-position="0.3" step-size="0.4">
+    <amp-img src="/1.jpg" layout="fill"></amp-img>
+    <amp-img src="/2.jpg" layout="fill"></amp-img>
+  </amp-image-slider>
+
+  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" initial-slider-position="1.5" step-size="1.5">
```

Comment:
It looks like this is intended to test it failing validation. Please put a comment before each test indicating what's expected and why and preferably group valid ones together before the invalid ones.

e.g.
```
  <!-- Valid -->
  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" disable-hint-reappear>
￼    <amp-img src="/1.jpg" layout="fill"></amp-img>
￼    <amp-img src="/2.jpg" layout="fill"></amp-img>
￼  </amp-image-slider>
  <!-- Invalid: initial-slider-position and step-size set out of bounds of acceptable values. -->
  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" initial-slider-position="1.5" step-size="1.5">
    <amp-img src="/1.jpg" layout="fill"></amp-img>
￼    <amp-img src="/2.jpg" layout="fill"></amp-img>
￼  </amp-image-slider>

Reply:
@honeybadgerdontcare Added comments. Thanks for pointing out!